README.md

```diff
@@ -1191,6 +1192,35 @@ f() // -> undefined
 
 You might expect `{}` instead of `undefined`. This is because the curly braces are part of the syntax of the arrow functions, so `f` will return undefined.
 
+## `arguments` and arrow functions
+
+Consider the example below:
+
+```js
+let f = function(){
+  return arguments;
+}
+f('a'); // -> ["a", callee: ƒ, ...]
```

Comment:
The `callee` arguments depends a lot from the vendor. It is deprecated but some some JS engines still return it.